src/compiler/data.ts

```diff
@@ -18,7 +19,7 @@ import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../type';
  *                 If the encoding contains aggregate value, this will also create
  *                 aggregate table as well.
  */
-export default function(model: Model) {
+export function def(model: Model) {
```

Comment:
I disagree that default is weird. It is in the es6 standard https://people.mozilla.org/~jorendorff/es6-draft.html#sec-exports. I think we shouldn't change this for the wrong reasons.